src/test/ui/proc-macro/issue-75930-derive-cfg.stdout

```diff
@@ -1,4 +1,4 @@
-PRINT-ATTR INPUT (DISPLAY): #[allow(dead_code)] #[derive(Print)] #[print_helper(b)] #[print_helper(a)]
+PRINT-ATTR INPUT (DISPLAY): #[print_helper(a)] #[allow(dead_code)] #[derive(Print)] #[print_helper(b)]
```

Comment:
This is a subtle detail that I just noticed - since we perform cfg-stripping before invoking custom attributes (e.g. `#[my_attr] #[cfg(FALSE)] struct Foo {}` does nothing), we also expand `#[cfg_attr]`. So, this attribute macro sees expanded `#[cfg_attr]` attributes (e.g. the `#[allow(dead_code)]` at the 'top level' of the attribute target, but not anywhere else. For example, you can see `#[cfg_attr(not(FALSE), allow(warnings))] ` in the input below.

Reply:
>since we perform cfg-stripping before invoking custom attributes

This is something I want to change, btw (and make the expansion order fully left-to-right).
It will certainly needs a crater run, but I think we'll be able to do this in practice.